src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/index.jelly

```diff
@@ -51,7 +51,7 @@ THE SOFTWARE.
                                     <j:set var="it" value="${item}"/>
                                     <j:set var="instance" value="${null}"/>
                                     <j:set var="help" value="${descriptor.helpFile}"/>
-                                    <f:entry title="&lt;strong&gt;&lt;code&gt;${symbol}&lt;code&gt;&lt;/strong&gt;" help="${help}" />
+                                    <f:entry title="STEP: ${symbol}" help="${help}" />
```

Comment:
Is the prefix even needed? / why is it capitalised? 

![image](https://user-images.githubusercontent.com/21194782/108111258-5d6ba480-708c-11eb-98b0-6c7051ab6bd3.png)

```suggestion
                                    <f:entry title="${symbol}" help="${help}" />
```